reqs/management/commands/import_reqs.py

```diff
@@ -93,6 +93,36 @@ def find_key(key_name, row):
     return keys[0]
 
 
+def handle_transposed_reqstatus(row):
+    """
+    There are a number of rows that have date content in the ``reqStatus``
+    field and non-date content in the ``policySunset`` field.
+
+    We're going to assume that this means they were transposed, and correct
+    those rows.
+    """
+    if "reqStatus" not in row:
+        return row
+
+    status_date, sunset_date = None, None
+    try:
+        status_date = dateutil_parser.parse(
+            row["reqStatus"]).strftime("%Y-%m-%d")
+    except ValueError:
+        pass  # We expect this to not be a date.
+    try:
+        status_date = dateutil_parser.parse(
+            row["policySunset"]).strftime("%Y-%m-%d")
```

Comment:
Yes, because the row values get passed to something later that converts them to dates, and this seemed like the better option than special handling only for rows that get this transposition done to them.